ui/webui/src/components/Error.jsx

```diff
@@ -192,11 +194,19 @@ const addExceptionDataToReportURL = (url, exception) => {
 const exceptionInfo = (exception, idPrefix) => {
     const exceptionNamePrefix = exception.name ? exception.name + ": " : "";
     return (
-        <TextContent id={idPrefix + "-bz-report-modal-details"}>
-            <Text component={TextVariants.p}>
-                {exceptionNamePrefix + exception.message}
-            </Text>
-        </TextContent>
+        <Stack hasGutter>
+            <TextContent id={idPrefix + "-bz-report-modal-details"}>
+                <Text component={TextVariants.p}>
+                    {exceptionNamePrefix + exception.message}
+                </Text>
+            </TextContent>
+            {exception.stack &&
```

Comment:
I know this was cherry-picked from the commit that I wrote, but as said on slack, I think we do not need to show  the stracktrace inside the error modal. 
The reason we show the log from journal is to allow the user to edit it so as to remove sensitive information like passwords etc. 

So I think it's enough to store the stackstrace into a file, like webui-js-stacktrace.txt, and that we inform the user about the existence of it.

The design can be discussed with @garrett

Reply:
I'll definitely edit it again, I honestly didn't really understand the messages I have to do here, so I at least got the existing solution to work in all cases. Now I get it and I'll edit it

Reply:
I agree with @KKoukiou about not showing the stacktrace. And we wouldn't show the log, except that it might have some user identifiable parts (although even this is somewhat questionable, as it's on a live system without user data... I suppose a created user or IP address _might_ be shown? I can't think of much else that would be user-identifiable in the logs.)

Ideally, we'd just show a basic summary. People encountering errors can't really do anything about them for the most part.